### PR TITLE
Include DB_CONN in .env

### PR DESCRIPTION
**Pull request by sethvargo**
**Release Note**

```release-note
NONE
```

### CODE REVIEW

File: terraform/main.tf [hunk 1]
```diff
@@ -148,11 +148,13 @@ export COOKIE_KEYS="secret://${google_secret_manager_secret_version.cookie-hmac-
 # Note: these configurations assume you're using the Cloud SQL proxy!
 export DB_APIKEY_DATABASE_KEY="secret://${google_secret_manager_secret_version.db-apikey-db-hmac.id}"
 export DB_APIKEY_SIGNATURE_KEY="secret://${google_secret_manager_secret_version.db-apikey-sig-hmac.id}"
+export DB_CONN="${google_sql_database_instance.db-inst.connection_name}"
+export DB_DEBUG="true"
```
Comment (mikehelmick) [inline terraform/main.tf:152]: not sure we want this true by default

Comment (icco) [reply terraform/main.tf:152]: I mean, this is just for the users local config, so I think it's good?

Comment (sethvargo) [reply terraform/main.tf:152]: Yea, this would be local config (primarily used when running migrations). This is _not_ set on the service.

**Pull request merged**